examples/jenkins/pipeline/samplepipeline.json

```diff
@@ -2,12 +2,12 @@
   "kind": "Template",
   "apiVersion": "v1",
   "metadata": {
-    "name": "ruby-helloworld-sample",
+    "name": "sample-pipeline",
     "creationTimestamp": null,
     "annotations": {
-      "description": "This example shows how to create a simple ruby application in openshift origin v3",
+      "description": "This example uses a pipeline to build and deploy a simple ruby application in openshift origin v3",
```

Comment:
the template isn't in the sample app, it's actually coded into the template.

"This example showcases the new Jenkins Pipeline integration in OpenShift to perform continuous integration and deployment right from the platform. Within the sample is a Jenkinsfile - a definition of a multi-stage CI process that leverages the underlying OpenShift platform for dynamic and scalable builds. OpenShift integrates the status of your pipeline builds into the web console allowing you to see your entire application lifecycle."

What did you have in mind for the "for more information" reference?  official docs?  more description of this specific sample (the readme)?